sci-libs/gdal/gdal1-1.11.5.recipe

```diff
@@ -116,26 +124,73 @@ BUILD_PREREQUIRES="
 	cmd:pkg_config$secondaryArchSuffix
 	"
 
-defineDebugInfoPackage gdal1$secondaryArchSuffix \
-	$libDir/libgdal.so.$libVersion
+debugList=("$libDir"/libgdal.so.$libVersion)
+if [ -z "$commandSuffix" ]; then
+debugList+=(
+	"$commandBinDir"/gdal_contour \
+	"$commandBinDir"/gdal_grid \
+	"$commandBinDir"/gdal_rasterize \
+	"$commandBinDir"/gdal_translate \
+	"$commandBinDir"/gdaladdo \
+	"$commandBinDir"/gdalbuildvrt \
+	"$commandBinDir"/gdaldem \
+	"$commandBinDir"/gdalenhance \
+	"$commandBinDir"/gdalinfo \
+	"$commandBinDir"/gdallocationinfo \
+	"$commandBinDir"/gdalmanage \
+	"$commandBinDir"/gdalserver \
+	"$commandBinDir"/gdalsrsinfo \
+	"$commandBinDir"/gdaltindex \
+	"$commandBinDir"/gdaltransform \
+	"$commandBinDir"/gdalwarp \
+	"$commandBinDir"/nearblack \
+	"$commandBinDir"/ogr2ogr \
+	"$commandBinDir"/ogrinfo \
+	"$commandBinDir"/ogrlineref \
+	"$commandBinDir"/ogrtindex \
+	"$commandBinDir"/testepsg
+	)
+fi
+defineDebugInfoPackage gdal1$secondaryArchSuffix "${debugList[@]}"
 
 BUILD()
 {
 	expatDir=$portPackageLinksDir/lib~libexpat
 	autoreconf -fi
-	runConfigure ./configure --with-expat=$expatDir
+	runConfigure --omit-dirs "binDir dataDir" ./configure \
+		--bindir "$commandBinDir" \
+		--datadir="$dataDir"/gdal1 \
```

Comment:
Hmm, we forgot to add a few `CONFLICTS`. So here are my suggestions:
```diff
 	lib:libtiff$secondaryArchSuffix
 	lib:libz$secondaryArchSuffix
 	"
+CONFLICTS="
+	gdal$secondaryArchSuffix
+	"
 
 PROVIDES_devel="
```
```diff
 REQUIRES_devel="
 	gdal1$secondaryArchSuffix == $portVersion base
 	"
+CONFLICTS_devel="
+	gdal${secondaryArchSuffix}_devel
+	"
 
 BUILD_REQUIRES="
```
```diff
-		--datadir="$dataDir"/gdal1 \
+		--datadir="$dataDir"/gdal \
```

Reply:
My mistake. We don't need a `CONFLICTS` for the base package because the lib's soname is not the same as the one from the newer gdal recipes.